package/src/TransactionContext/initialState.tsx

```diff
@@ -1,46 +1,54 @@
-import { StateType } from "./models";
 import { DEFAULTS as defaultSettings } from "layer2";
+import { StateType } from "./models";
 
-export default {
+const initialState: StateType = {
   key: 0,
   userId: "",
-  currentQuote: {
-    blockNumber: "0",
-    amount: "0",
-    amountDecimals: "0",
-    quote: "0",
-    quoteDecimals: "0",
-    quoteGasAdjusted: "0",
-    quoteGasAdjustedDecimals: "0",
-    gasUseEstimateQuote: "0",
-    gasUseEstimateQuoteDecimals: "0",
-    gasUseEstimate: "0",
-    gasUseEstimateUSD: "0",
-    gasPriceWei: "0",
+  inAmount: 0,
+  txId: "---transaction--id---",
+  quote: {
+    blockNumber: "",
+    amount: "",
+    amountDecimals: "",
+    quote: "",
+    quoteDecimals: "",
+    quoteGasAdjusted: "",
+    quoteGasAdjustedDecimals: "",
+    gasUseEstimateQuote: "",
+    gasUseEstimateQuoteDecimals: "",
+    gasUseEstimate: "",
+    gasUseEstimateUSD: "",
+    gasPriceWei: "",
     route: [],
     routeString: "",
-    quoteId: "0",
+    quoteId: "",
   },
   tokenIn: {
     name: "",
     address: "",
     symbol: "",
     decimals: 18,
-    chainId: 4,
+    chainId: 3,
```

Comment:
Would this have to be manually update before going live?

Reply:
I think a safer default value would be testnet. And so only if we actually pull in a mainnet token from the BE response do we overwrite the test-env-first value.